src/test/java/seedu/findvisor/storage/JsonAdaptedPersonTest.java

```diff
@@ -122,11 +125,22 @@ public void toModelType_nullMeeting_throwsIllegalValueException() {
     }
 
     @Test
-    public void toModelType_invalidMeeting_throwsIllegalValueException() {
-        JsonAdaptedMeeting invalidMeeting = new JsonAdaptedMeeting("INVALID_START", "INVALID_END");
+    public void toModelType_invalidMeetingDateTime_throwsIllegalValueException() {
+        JsonAdaptedMeeting invalidMeeting = new JsonAdaptedMeeting("INVALID_START",
+                "INVALID_END", INVALID_MEETING_REMARK);
```

Comment:
Just a nit, but would it be better to use a valid meeting remark for this test since the test for invalid meeting datetime?

Reply:
Yeap good suggestion